From da5e5ca98090f600ed65b476e2d6c3995c611191 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:41 -0400
Subject: [redhat] kabi: Adding symbol blk_make_request (block/blk-core.c)

Message-id: <3931d652658d96af5f784ea639058edc30eab75b.1410510773.git.poros@redhat.com>
Patchwork-id: 95914
O-Subject: [PATCH RHEL7.1 041/233] kabi: Adding symbol blk_make_request (block/blk-core.c)
Bugzilla: 1071098
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Given a bio, allocate a corresponding struct request.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_make_request
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_make_request
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_make_request

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_make_request b/redhat/kabi/kabi-module/kabi_ppc64/blk_make_request
new file mode 100644
index 0000000..fbb36c7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_make_request
@@ -0,0 +1,2 @@
+#1-
+0x226a4d31 blk_make_request vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_make_request b/redhat/kabi/kabi-module/kabi_s390x/blk_make_request
new file mode 100644
index 0000000..c8a2bbe
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_make_request
@@ -0,0 +1,2 @@
+#1-
+0xa579bb8d blk_make_request vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_make_request b/redhat/kabi/kabi-module/kabi_x86_64/blk_make_request
new file mode 100644
index 0000000..2b3b81a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_make_request
@@ -0,0 +1,2 @@
+#1-
+0x449b3dbe blk_make_request vmlinux EXPORT_SYMBOL
-- 
1.7.1